### ISSUE TITLE
Remote files not compiled

### ISSUE THREAD

**Issue by hedii** [bdkjones/CodeKit2#583]
Hi,
I have mount my remote website with transmit as a local drive.
This folder is my project in codekit.
When i make a modification on the remote file (eg: i edit a js file), I see codekit refreshing, but not compiling the file.
i have to manually push the button 'process' to run the compilation.
I want codekit to compile the remote file, is there a workaround?

**Comment by bdkjones** [comment]
Hi Heidi,

```
Please record a short screencast showing me the issue. Be sure to include the selected file in CodeKit so I can see the options you’ve set for that file. Thanks
```

**Comment by hedii** [comment]
the screencast: https://www.youtube.com/watch?v=mqRWOip3nrg (the first video i uploaded didn't show the right file)

when save the file, it is not compiled, when i click 'process' it is compiled (the notification is visible in one of my other screens).

i try to put codekit config file here, but github does not accept it

**Comment by hedii** [comment]
here the codekit config file : https://gist.github.com/hedii/719108ec8e171f860eee

**Comment by hedii** [comment]
hi @bdkjones 
have you seen the infos i posted?
i still have the same problem, even with the last update
codekit 2.5.1
osx 10.11.1
folder mounted remotely via transmit disk latest version

**Comment by filipjnc** [comment]
Hi everyone,
I encountered the same issue with CodeKit. It seems that it does recognize file changes on remote disks, however it doesn't proceed to compiling them. Is a fix awaited soon? Thank you!

**Comment by hedii** [comment]
Don't know, no news on that issue.
Honestly i stopped using codekit because of that. I'm now using gulp...

**Comment by filipjnc** [comment]
I really like CodeKit and I don't feel comfortable moving over to another Toolbox :(

**Comment by bdkjones** [comment]
I’m looking into the issue. Kernel-level file events are way more complicated than you’d think.

**Comment by filipjnc** [comment]
@bdkjones I appreciate that. 

Btw, I only seem to have this issue with Coda. When saving files with Sublime Text, CodeKit does compile them.

**Comment by bdkjones** [comment]
Yep. When you save files with Sublime Text, are you mounting the remote disk using Transmit 4? Or do you mean that when you save with Sublime, the files are local to your machine?

**Comment by filipjnc** [comment]
In both cases I'm mounting the remote disk using Transmit 4. Could it be a Coda issue?
